lib/dialects/sqlite3/schema/internal/sqlite-ddl-operations.js

```diff
@@ -13,20 +13,14 @@ function insertChunked(trx, chunkSize, target, iterator, existingData) {
   return result;
 }
 
-function createTempTable(createTable, tablename, alteredName) {
-  return createTable.sql.replace(tablename, alteredName);
-}
-
-// ToDo To be removed
-async function copyData(trx, tableName, alteredName) {
-  const existingData = await trx.raw(`SELECT * FROM "${tableName}"`);
-  return insertChunked(trx, 20, alteredName, identity, existingData);
+function createNewTable(sql, tablename, alteredName) {
+  return sql.replace(tablename, alteredName);
 }
 
 // ToDo To be removed
 async function reinsertData(trx, iterator, tableName, alteredName) {
```

Comment:
Is the name still accurate, if we are using it not for reinsertion, but just for copying?

Reply:
Not precisely. I can rename the function to `copyData`. But wouldn't that mean that `reinsertMapped` and `generateReinsertCommands` aren't accurate either? Any suggestions for alternative names?

Reply:
How about `copyMapped` and `generateCopyDataCommands`?

Reply:
I thought of that as well but those methods are doing much more than just copying, especially with PR #4189. What do you think of `alter` (or `alterMapped`) and `generateAlterCommands`?

Reply:
Sure!